### DISCUSSION TITLE
Integrate with Facebook Products

### DISCUSSION THREAD

**Discussion by Kashkin** [craftcms/commerce#2315] [Ideas]
### Description
Popular ecommerce platforms like Shopify, Magento and WooCommerce integrate directly with Facebook Products, allowing for products to automatically appear in Facebook's shop section (see [this help topic](https://www.facebook.com/business/help/646757258759189?helpref=faq_content)).

Could this functionality be achieved in Craft Commerce? Do you think this will ever be part of core, or is it better suited as a plugin?

**Comment by lukeholder** [comment]
Sounds like a plugin, but will leave it on the list.

**Comment by sjcallender** [comment]
@Kashkin We have this plugin created but not publicly available yet. I'll reply here when it's available. Or DM me.

**Comment by Marcoevich** [comment]
Hi @sjcallender I'm curious is this plugin public already? We could certainly use functionality like this.

**Comment by nfourtythree** [comment]
Hi @Marcoevich 

There is a [Facebook Product Catalog](https://plugins.craftcms.com/facebook-catalog) plugin available in the plugin store.

**Comment by Marcoevich** [comment]
Hi @nfourtythree, this looks like exactly what we need.

Thanks!
